src/canopy_cli/cli.py

```diff
@@ -200,18 +207,18 @@ def health(url):
 
 @cli.command(
     help=(
-        """
-        \b
-        Create a new Pinecone index that will be used by Canopy.
-
-        A Canopy service cannot be started without a Pinecone index that is configured
-        to work with Canopy. This command creates a new Pinecone index and configures
-        it in the right schema.
-
-        If the embedding vectors' dimension is not explicitly configured in
-        the config file, the embedding model will be tapped with a single token to
-        infer the dimensionality of the embedding space.
-        """  # noqa: E501
+            """
```

Comment:
Please don't push these code style changes as part of this commit. It can only lead to conflicts with other open PRs.  
Code style changes (to lines unaffected by the commit's purpose) can be done via a separate dedicated PR